common/djangoapps/student/views.py

```diff
@@ -1804,6 +1805,30 @@ def create_account_with_params(request, params):
     return new_user
 
 
+def _enroll_user_in_pending_courses(student):
+    """
+    Enroll student in any pending courses he/she may have.
+    """
+    if not student:
+        log.error(u"There is no student")
+        return
+    else:
+        ceas = CourseEnrollmentAllowed.objects.filter(email=student.email)
+        for cea in ceas:
+            if cea.auto_enroll:
+                enrollment = CourseEnrollment.enroll(student, cea.course_id)
+                manual_enrollment_audit = ManualEnrollmentAudit.get_manual_enrollment_by_email(student.email)
+                if manual_enrollment_audit is not None:
+                    # get the enrolled by user and reason from the ManualEnrollmentAudit table.
+                    # then create a new ManualEnrollmentAudit table entry for the same email
+                    # different transition state.
+                    ManualEnrollmentAudit.create_manual_enrollment_audit(
+                        manual_enrollment_audit.enrolled_by, student.email, ALLOWEDTOENROLL_TO_ENROLLED,
+                        manual_enrollment_audit.reason, enrollment
+                    )
+    return student
```

Comment:
we're not using the output of this function anywhere — we're only concerned about the side effects of `_enroll_user_in_pending_courses`. When it's used elsewhere, its output isn't assigned to a variable.